security/libprelude/Portfile

```diff
@@ -54,16 +62,18 @@ destroot.keepdirs \
         ${destroot}${prefix}/var/spool/prelude \
         ${destroot}${prefix}/etc/prelude/profile
 
-post-destroot {
-    delete ${destroot}${prefix}/lib/charset.alias
-}
-
 variant python27 description {python2.7 bindings} {
-        configure.args-delete	--without-python
-        configure.args-append	--with-python=${prefix}/bin/python2.7
+        configure.args-delete	--without-python2
+        configure.args-append	--with-python2=${prefix}/bin/python2.7
         depends_lib-append port:python27
 }
 
+variant python34 description {python3.4 bindings} {
```

Comment:
please don't add a Python 3.4 variant, as that version is already EOL - instead please use Python 3.8 here.

Reply:
ok, done